### PR TITLE
Tweaks from SB2 importer audit

### PR DESCRIPTION
**Pull request by tmickel**
Added:
- some null checks
- extra math_\* types
- converts colors to hex on import
- various JSON tweaks to make consistent with scratch-blocks

### CODE REVIEW

**Comment by TheBrokenRail** [comment]
The Variable fix should make a lot of imported projects more understandable! Hooray!

File: src/util/math-util.js [hunk 1]
```diff
@@ -45,4 +45,15 @@ MathUtil.wrapClamp = function (n, min, max) {
     return n - Math.floor((n - min) / range) * range;
 };
 
+/**
+ * Convert a Scratch color number to a hex string, #RRGGBB.
+ * @param {number} color RGB color as a decimal.
+ * @return {string} RGB color as #RRGGBB hex string.
+ */
+MathUtil.scratchColorToHex = function (color) {
```
Comment (thisandagain) [inline src/util/math-util.js]: I wonder if we might want to move this into a dedicated `color` utility module eventually. This should also have test coverage.

**Comment by thisandagain** [comment]
Looks good. One minor comment.

File: src/util/math-util.js (hunk 1 above)
Comment (tmickel) [reply src/util/math-util.js]: sounds good - seems like we'll want more in the future.

**Pull request merged**